sagemaker_neo_compilation_jobs/mxnet_mnist/input.html

```diff
@@ -1,270 +0,0 @@
-<script type="text/Javascript">
```

Comment:
should we consider leaving this here, and adding a separate python script called "get_input.py" that does what the notebook used to do? And add something that saves the data as *.npy file? Then if people wanna go generate new inputs they can... just a thought.

Reply:
Thanks for taking this up offline. We are now good to merge it.